tests/test_dataset_list.py

```diff
@@ -0,0 +1,42 @@
+from unittest import TestCase
+
+from datasets import Sequence, Value
+from datasets.arrow_dataset import Dataset
+
+
+class DatasetListTest(TestCase):
```

Comment:
You can put these tests in `test_arrow_dataset.py` with the `from_dict` tests :)

Reply:
A bit confused about this, since there is also "test_dataset_dict.py"

Reply:
oh right ! This way is fine then sorry

Reply:
Alright, I think it's all set then :)